podkrepi-bg/frontend
# [Feature] Add see more button

## Motivation and context

As the members text was too long in order to improve the UX and UI we added See more/less buttons
issue
https://github.com/podkrepi-bg/frontend/issues/1091

## Screenshots:

| Before              | After              |
| ------------------- | ------------------ |
| [before screenshot] | [after screenshot] |

## Testing

1. go to the about page
2. click see  the button should have see more/less (Вижте повече/помалко) depending on the language of choice  
3.  test to see if changes properly 

![Screenshot_20221120_115930](https://user-images.githubusercontent.com/63183508/202915208-bb36ddd4-63e1-4b3f-8362-9dfd2a34624a.png)
![Screenshot_20221120_120027](https://user-images.githubusercontent.com/63183508/202915233-98dfae94-2073-4ef0-ac6d-21313036769e.png)
